parse.py

```diff
@@ -508,7 +508,7 @@ def create_inst_dict(file_filter, include_pseudo=False, include_pseudo_ops=[]):
 def make_priv_latex_table():
     latex_file = open("priv-instr-table.tex", "w")
     type_list = ["R-type", "I-type"]
-    system_instr = ["_h", "_s", "_system", "_svinval", "64_h"]
+    system_instr = ["_h", "_s", "_system", "_svinval", "64_h", "svinval_h"]
```

Comment:
@huxuan0307 and @aswaterman , I suspect as this is because it is expecting the `_` (underscore) before `svinval_h` to detect Hypervisor Instructions for `Svinval`.
can you try this out @huxuan0307  and see if it passes.